scripts/yocto-build-and-test.sh

```diff
@@ -523,11 +523,10 @@ build_and_test_client() {
             egrep -q '\bmender-image-uefi\b' <<<${features} && extension="uefiimg"
             egrep -q '\bmender-image-sd\b' <<<${features} && extension="sdimg"
 
-            copy_clean_image "${machine_name}" "${board_name}" "${image_name}" "${device_type}" "$extension"
-            copy_clean_image "${machine_name}" "${board_name}" "mender-image-full-cmdline-rofs" "${device_type}" "$extension"
-            copy_clean_image "${machine_name}" "${board_name}" "mender-image-full-cmdline-rofs-commercial" "${device_type}" "$extension"
-            copy_clean_image "${machine_name}" "${board_name}" "mender-monitor-image-full-cmdline" "${device_type}" "$extension"
-            copy_clean_image "${machine_name}" "${board_name}" "mender-gateway-image-full-cmdline" "${device_type}" "$extension"
+            for image_name in $images_to_build; do
```

Comment:
There is a bug here.

By reusing `image_name` variable it gets set to `mender-gateway-image-full-cmdline` after the loop and then [line 537](https://github.com/mendersoftware/mender-qa/pull/637/files#diff-7031fcd5c934f225781febe446a34594a9a36b335c540379c0dd376a8f06ba28R537) sets the wrong filename :boom: 

So it was better to use `img` after all :wink:

Reply:
hahahha, omfg :cricket:

Reply:
Fixed!